packages/lodestar-cli/src/options/beaconNodeOptions/network.ts

```diff
@@ -27,13 +27,13 @@ export const networkOptions = {
     group: "network",
   } as Options,
 
-  "network.bootnodes": {
+  "network.bootMultiaddrs": {
     type: "array",
     defaultDescription: JSON.stringify(defaultOptions.network.bootnodes),
     group: "network",
   } as Options,
 
-  "network.multiaddrs": {
+  "network.localMultiaddrs": {
```

Comment:
The property names here cannot be changed, they must match the actual BeaconNodeOption types.

Reply:
changes reverted